tests/playwright/package.json

```diff
@@ -10,7 +10,8 @@
     "test:ui": "npx playwright test ui-tests/* --project=chromium --project=firefox --project=webkit --project=\"Microsoft Edge\"",
     "test:ci": "npx playwright test accessibility-tests/* ui-tests/* --project=chromium --project=firefox --project=webkit --project=\"Microsoft Edge\"",
     "test:deployment": "npx playwright test --project=\"deployment-tests\"",
-    "test:integration": "npx playwright test --project=\"integration-tests\""
+    "test:integration": "npx playwright test --project=\"integration-tests\"",
+    "test:ui:trace": "npx playwright test ui-tests/* --project=chromium --project=firefox --project=webkit --project=\"Microsoft Edge\" --trace=on"
```

Comment:
This is a cool idea and really useful for local development. I think it would be good to only use one browser though - trace on makes tests take longer and especially in local dev we don't want to be taking longer than needed